packages/varo/src/Utils.js

```diff
@@ -1,13 +1,102 @@
 export function parseMetadata(message) {
-  if (!message.metadata?.content) {
+  const content = message.metadata?.content;
+  if (!content) {
     console.error('No metadata supplied.');
     return null;
   }
 
   try {
-    return JSON.parse(message.metadata.content);
+    return JSON.parse(content);
   } catch (error) {
     console.error('Invalid metadata JSON.', error);
     return null;
   }
 }
+
+// These report properties are also allowed in the record,
+// used for overriding report characteristics per record:
+// EDOP, EMSV, LACC, LAT, LNG, LSV, SIGN
+
+const defs = {
```

Comment:
Should this stuff be in job code, rather than adaptor code? Is this implementation specific?